pyproject.toml

```diff
@@ -39,10 +39,10 @@ classifiers = [
 ]
 
 [project.urls]
-homepage = "https://github.com/NyanKiyoshi/requests-hardened/"
-source = "https://github.com/NyanKiyoshi/requests-hardened/"
-issues = "https://github.com/NyanKiyoshi/requests-hardened/issues"
-changelog = "https://github.com/NyanKiyoshi/requests-hardened/releases/"
+Homepage = "https://github.com/saleor/requests-hardened/"
+Source = "https://github.com/saleor/requests-hardened/"
+Issues = "https://github.com/saleor/requests-hardened/issues"
+Changelog = "https://github.com/saleor/requests-hardened/releases/"
```

Comment:
The "Title Case" makes it more consistent with PyPI UI, and other projects, e.g.:

<img height="500" alt="Screen Shot 2023-09-05 at 7 28 13 AM" src="https://github.com/saleor/requests-hardened/assets/6186720/a76bd5b5-c86b-4680-9f9b-972d3ff4662f">
<img height="500" alt="Screen Shot 2023-09-05 at 7 28 20 AM" src="https://github.com/saleor/requests-hardened/assets/6186720/631c1e31-ef55-48ba-b8bc-17563b6e3fcd">